package.json

```diff
@@ -69,6 +69,8 @@
   "dependencies": {
     "detect-indent": "^7.0.1",
     "detect-newline": "^4.0.0",
+    "get-stdin": "^9.0.0",
+    "get-stream": "^6.0.1",
```

Comment:
This one should be a devDependency.

Reply:
done [keithamus/sort-package-json@`26c902f` (#296)](https://github.com/keithamus/sort-package-json/pull/296/commits/26c902f573e2c192bfd987eb75679efc6a8beba2)